proto/frequenz/api/microgrid/common.proto

```diff
@@ -145,91 +148,94 @@ message AC {
   // AC metrics of a single phase.
   message ACPhase {
     // The AC voltage between the line and the neutral wire.
-    // In Volts (V).
+    // In Volt (V).
     Metric voltage = 1;
 
     // AC current.
     // +ve current means consumption, away from the grid.
     // -ve current means supply into the grid.
-    // In Amperes (A).
+    // In Ampere (A).
     Metric current = 2;
 
     // AC active power.
     // +ve power means consumption, away from the grid.
     // -ve power means supply into the grid.
-    // In Watts (W).
+    // In Watt (W).
     Metric power_active = 3;
 
     // AC reactive power.
     // +ve power means inductive (leading).
     // -ve power means capacitive (lagging).
-    // In VArh.
+    // In Volt-Ampere reactive hour (VArh).
```

Comment:
The old doc was incorrect. It should be just `Volt-Ampere reactive`.

In general, power units do not have a time component in them, e.g. it's `Watt`, and not `Watt-hour`.

Reply:
Fixed